Added support for InvocationTransaction/GetScript

Taconut opened the pull request:
**What current issue(s) from Github does this address?**
Fixes CityOfZion/neo-python#324

**What problem does this PR solve?**
neo-project/neo has an implementation of InvocationTransaction.GetScript, but neither this nor neo-python contain it.

**How did you solve this problem?**
I created a new InvocationTransaction file and added GetScript.

**How did you make sure your solution works?**
I verified that the InvocationTransaction.py file conforms to the format of the other files in /interop/Neo/

localhuman commented on boa/interop/Neo/InvocationTransaction.py:
```diff
@@ -0,0 +1,19 @@
+class InvocationTransaction:
+
+    @property
+    def GetScript(self):
```
I would change this to the following so people can use it like so: `myscript = tx.Script`

```
@property
def Script(self)
```

localhuman commented:
Thanks for the contribution! 🎉 

To get the travis build to pass, you will need to merge the most recent changes in development into this branch.

Also, make sure to run `pycodestyle boa` and fix any stlye issues that you find.

Taconut commented:
Thay should take care of everything. I renamed the getter to "Script", removed the blank line at the end of the file, and pulled the recent changed from development into my branch.

localhuman commented:
⭐️
